pkg/crc/machine/delete.go

```diff
@@ -18,6 +18,13 @@ func (client *client) Delete() error {
 		return errors.Wrap(err, "Cannot remove machine")
 	}
 
+	// In case usermode networking make sure all the port bind on host should be released
+	if vm.vsock {
+		if err := unExposePorts(); err != nil {
```

Comment:
I tired unexposePorts before but it was showing as lint error in IDE so I need to use this way.

Reply:
I'm not getting any errors with `make cross-lint` after making this change to the dev branch
```
diff --git a/pkg/crc/machine/delete.go b/pkg/crc/machine/delete.go
index 2682efc4..9689c159 100644
--- a/pkg/crc/machine/delete.go
+++ b/pkg/crc/machine/delete.go
@@ -7,7 +7,12 @@ import (
 	"github.com/pkg/errors"
 )
 
+func unexposePort() error {
+	return nil
+}
+
 func (client *client) Delete() error {
+	_ = unexposePort()
 	vm, err := loadVirtualMachine(client.name, client.useVSock())
 	if err != nil && !errors.Is(err, errInvalidBundleMetadata) {
 		return errors.Wrap(err, "Cannot load machine")
```
Not sure what was failing on your side? If it's an IDE specific lint, maybe it can be fixed there/ignored?

Reply:
It was IDE specific not `make cross-lint` side, need to ignore it.